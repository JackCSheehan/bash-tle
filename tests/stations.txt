AEROCUBE 12A            
1 43556U 18046C   22257.29976014  .00022692  00000+0  52641-3 0  9999
2 43556  51.6354 109.8054 0005697   3.0795 357.0225 15.42281973233012
AEROCUBE 12B            
1 43557U 18046D   22256.97717455  .00019746  00000+0  50152-3 0  9997
2 43557  51.6361 117.3198 0006047 358.2894   1.8069 15.39624388232835
ISS DEB                 
1 47853U 98067RZ  22257.37467644  .00012377  00000+0  18683-3 0  9994
2 47853  51.6401 246.8753 0002140   2.2968 357.8032 15.55156353 85746
MMSATS-1                
1 47976U 98067SJ  22257.77749356  .00035046  00000+0  38349-3 0  9996
2 47976  51.6386 235.1169 0002352  38.7093  23.9344 15.62947322 84214
CSS (TIANHE)            
1 48274U 21035A   22257.75742604  .00024104  00000+0  26718-3 0  9996
2 48274  41.4736  33.2329 0006061 314.0267 183.7050 15.62374121 78775