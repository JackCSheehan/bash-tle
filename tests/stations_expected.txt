AEROCUBE 12A,43556,U,18,046,C,22,257.29976014,.00022692,00000+0,52641-3,0,999,9,43556,51.6354,109.8054,0005697,3.0795,357.0225,15.42281973,23301,2
AEROCUBE 12B,43557,U,18,046,D,22,256.97717455,.00019746,00000+0,50152-3,0,999,7,43557,51.6361,117.3198,0006047,358.2894,1.8069,15.39624388,23283,5
ISS DEB,47853,U,98,067,RZ,22,257.37467644,.00012377,00000+0,18683-3,0,999,4,47853,51.6401,246.8753,0002140,2.2968,357.8032,15.55156353,8574,6
MMSATS-1,47976,U,98,067,SJ,22,257.77749356,.00035046,00000+0,38349-3,0,999,6,47976,51.6386,235.1169,0002352,38.7093,23.9344,15.62947322,8421,4
CSS (TIANHE),48274,U,21,035,A,22,257.75742604,.00024104,00000+0,26718-3,0,999,6,48274,41.4736,33.2329,0006061,314.0267,183.7050,15.62374121,7877,5
